warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__pathways_valid.sql

```diff
@@ -57,30 +30,88 @@ validation_fact_daily_feed_codes_pathway_related AS (
             )
 ),
 
-pathway_validation_notices_by_day AS (
+feed_has_rail AS (
+    SELECT
+        feed_key,
+        LOGICAL_OR(route_type = "2") AS has_rail
```

Comment:
nit: could you link/comment spec

Reply:
I did but I also would argue that the GTFS schedule spec is implicitly referenced throughout the codebase... Not sure what the balance is for this.